app/models/subscriber.rb

```diff
@@ -0,0 +1,32 @@
+class Subscriber
```

Comment:
my perspective:

* **models**: store state somewhere (in the DB, a file, or even in-memory)
* **services (or interactors, jobs, whatever)**: perform a task, but don't store any state, unless they need it while performing said task

I guess the two concepts overlap a bit, but I still like to keep them apart